### PR TITLE
Add script to check SDL does not resend OnSeekMediaClockTimer if seek…

### PR DESCRIPTION
**Pull request by OVikhrov**
…Time minutes out lower bound
@aarotar please review

### CODE REVIEW

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_minutes_Out_Lower_Bound.lua [hunk 1]
```diff
@@ -0,0 +1,94 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_minutes_Out_Lower_Bound.lua]: @OVikhrov, please update Requirement summary with related **functional** requirement also

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_minutes_Out_Lower_Bound.lua [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+function Test:Preconditions_ActivateApp()
+commonSteps:ActivateAppInSpecificLevel(self, self.applications["Test Application"])
+EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_OnSeekMediaClockTimer_minutes_Value_Of_seekTime_Is_Out_Lower_Bound()
+ self.hmiConnection:SendNotification("UI.OnSeekMediaClockTimer",{seekTime =  {hours = 3, minutes = -1, seconds = 33}, appID = self.applications["Test Application"]})
+
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_minutes_Out_Lower_Bound.lua]: @OVikhrov, plase avoid redundant empty lines

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_minutes_Out_Lower_Bound.lua (hunk 1 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_minutes_Out_Lower_Bound.lua]: Accepted

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_minutes_Out_Lower_Bound.lua (hunk 2 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_minutes_Out_Lower_Bound.lua]: Accepted

**Comment by OVikhrov** [comment]
@aarotar Plese review iteration#2

**Pull request merged**